### ISSUE TITLE
Cannot search on values with an &

### ISSUE THREAD

**Issue by aviking** [tsgrp/HPI#1278]
When executing a search if a search value contains an "&" the value get's truncated. Example: searching for "Labels & Values" will execute a search for "Labels ". See Screen cam below. Another issue is if you are given a search link and past it into a new browser window, the same issue occurs even though the "&" is URL encoded.
Talk to Joe H before starting.

![searchampersandissue](https://cloud.githubusercontent.com/assets/2895070/21649888/c8d4c664-d268-11e6-85f1-1ffc2172d01e.gif)

**Comment by gsteimer** [comment]
Remember to test a saved search with an `&` in a param

**Comment by grahamsinger** [comment]
Fixed with revision 16947.
